server/db/flashcards/create.js

```diff
@@ -0,0 +1,15 @@
+module.exports = knex => async (params) => {
+  const flashcard = params;
```

Comment:
please rename params to flashcards. than you don't need extra an additional const